woocommerce-paypal-payments.php

```diff
@@ -24,6 +24,8 @@
 define( 'PAYPAL_INTEGRATION_DATE', '2022-04-13' );
 
 define( 'PPCP_FLAG_SUBSCRIPTION', true );
+define( 'PPCP_FLAG_OXXO', apply_filters( 'woocommerce_paypal_payments_enable_oxxo', false ) );
+define( 'PPCP_FLAG_SEPARATE_APM_BUTTONS', apply_filters( 'woocommerce_paypal_payments_enable_separate_apm_buttons', false ) );
```

Comment:
these filters should have some more specific name, like `..._feature`. Or `allow` instead of `enable` (but it's already used for cards).

Otherwise it sounds like they enable the gateways, like the checkbox in WC settings.